.github/workflows/php-ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Wikidata Reference Hunt Game
+
+on: 
+  push:
+    paths:
+      - 'wikidata_game/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Since WMF infra is all debian. Can we change it (if it supports it) to debian-latest or buster?

Reply:
Unfortunately not (unless we host our own image to run this on [total overkill]). GitHub only hosts ubuntu for these runners: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners

Reply:
ugh, it's fine then.